connection.go

```diff
@@ -387,6 +385,14 @@ func (c *Connection) sendPacketRaw(pt uint8, connId ConnectionId, pn uint64, ver
 		}
 	}
 
+	return aead
+}
+
+func (c *Connection) sendPacketRaw(pt uint8, connId ConnectionId, pn uint64, version VersionNumber, payload []byte) error {
+	c.log(logTypeConnection, "Sending packet PT=%v PN=%x: %s", pt, c.nextSendPacket, dumpPacket(payload))
```

Comment:
You can use c.logPacket() here.

Reply:
Not sure what function you mean. I can not find the string `logPacket` anywhere in the source.

Reply:
Oh, I forgot to push. See now.